### DISCUSSION TITLE
Merge Confidence Badges for Bitbucket Cloud

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#16795] [Request Help] [answered]
Hi all

Got a question regarding the beta Merge Confidence badges feature.

According to the [supported platforms](https://docs.renovatebot.com/merge-confidence/#supported-platforms) documentation, it is available on any supported Renovate platform.

I've tried to enable this in our Bitbucket Cloud environment, but haven't had any luck with badges appearing on the PRs.

Works nicely in our GitHub space, though.

Any pointers would be appreciated

**Comment by rarkins** [comment]
Can you reproduce the problem in a public repo? Make sure you're using one of the supported ecosystems such as npm

**Comment by HonkingGoose** [comment]
Does BitBucket Cloud allow the `@` character in links?

You may be affected by this bug:

- https://github.com/whitesource/merge-confidence/issues/1641

**Reply by setchy** [reply]
Thanks @HonkingGoose - looks like Bitbucket Cloud is ok with this

**Comment by setchy** [comment, marked as answer]
Thanks for the input @rarkins @HonkingGoose .  

Looks like our nightly pipeline worked just fine., so a false alarm in the end.

My suspicion is during my local testing yesterday for some reason it wasn't fetching the latest shared preset configurations (cache perhaps?).

All good now, thank you both
